### PR TITLE
Fix broken ID.

### PR DESCRIPTION
**Pull request by cahrens**
[TNL-5786](https://openedx.atlassian.net/browse/TNL-5786)

Sandbox: https://a11y.sandbox.edx.org/courses/course-v1:edx+ORA203+course/courseware/57a3f9d51d424f6cb922f0d69cba868d/bb563abc989340d8806920902f267ca3/

Please review: @staubina and @cptvitamin

### CODE REVIEW

File: test/acceptance/accessibility.py [hunk 1]
```diff
@@ -17,23 +17,22 @@ def setUp(self, problem_type, staff=False):
...
+        self._configure_a11y_audit_config(page)
+        page.a11y_audit.check_for_accessibility_errors()
+
+    def _configure_a11y_audit_config(self, page):
         page.a11y_audit.config.set_scope(
             exclude=[
-                ".container-footer",
-                ".nav-skip",
-                "#global-navigation",
+                "#footer-edx-v3",  # Links to Facebook, Twitter, etc. Populated in production, but not here.
```
Comment (cahrens) [inline test/acceptance/accessibility.py]: Cleaned up ignores and excludes so we are covering as much as possible.

File: test/acceptance/accessibility.py [hunk 2]
```diff
@@ -282,6 +281,26 @@ def test_multiple_ora_complete_flow(self):
...
 
+class FileUploadA11yTest(OpenAssessmentA11yTest):
+
+    def setUp(self):
+        super(FileUploadA11yTest, self).setUp('file_upload')
+
+    def test_file_upload(self):
+        self.auto_auth_page.visit()
+        self.submission_page.visit()
+        self._check_a11y(self.submission_page)
```
Comment (cahrens) [inline test/acceptance/accessibility.py]: This test caught the issue being fixed.

File: test/acceptance/accessibility.py [hunk 3]
```diff
@@ -282,6 +281,26 @@ def test_multiple_ora_complete_flow(self):
...
+
+    def _configure_a11y_audit_config(self, page):
+        super(FileUploadA11yTest, self)._configure_a11y_audit_config(page)
+        page.a11y_audit.config.set_rules({
+            "ignore": [
+                "color-contrast",  # TODO: TNL-5104
+                "list",  # TODO: TNL-5900
```
Comment (cahrens) [inline test/acceptance/accessibility.py]: Problem with the list structure-- created TNL-5900, which is a subtask of the more general "list structure is crazy" ticket.

**Comment by cptvitamin** [comment]
👍

**Comment by cahrens** [comment]
@efischer19 I'm going to go ahead and merge since Mark reviewed. Thanks, @cptvitamin!

**Pull request merged**